api/controllers/DashboardController.js

```diff
@@ -16,275 +19,400 @@ var dashboardController = {
+                    // For each element in the database results array, add the application reference status
+                    // if one exists.
+
+                    for (let result of results) {
+                        const appStatus = appRef[result.unique_app_id];
+                        result.app_status =
+                            dashboardController._userFriendlyStatuses(
+                                appStatus,
+                                result.applicationtype
+                            );
+                        result.tracking_ref = trackRef[result.unique_app_id];
+                    }
+                }
+
+                const pageAttributes = {
+                    message: req.flash('info'),
+                    users_applications: results,
+                    moment,
+                    offset,
+                    sortOrder,
+                    paginationMessage,
+                    currentPage,
+                    totalPages,
+                    searchCriteria,
+                    user_data: userData,
+                    application_total: totalApplications,
+                };
+
+                return dashboardController._redirectToPage(
+                    pageAttributes,
+                    req,
+                    res
+                );
+            }
+        );
+    },
 
-                            var moment = require('moment');
-                            var pageUpperLimit = (offset + pageSize);
-                            if (pageUpperLimit > resultCount) {
-                                pageUpperLimit = resultCount;
-                            }
-                            var totalPages = resultCount%pageSize===0? resultCount / pageSize: Math.floor(resultCount / pageSize) + 1;
-                            var paginationMessage;
-                            if (resultCount === 0) {
-                                paginationMessage = 'No applications found';
-                            }
-                            else {
-                                paginationMessage = 'Showing ' + (offset + 1) + ' &ndash; ' + pageUpperLimit + ' of ' + resultCount + ' applications submitted in the last 60 days';
-                            }
+    _userFriendlyStatuses(casebookStatus, applicationtype) {
+        const eAppStatuses = {
+            Done: {
+                text: 'Completed',
+                colorClass: '', // dark blue
+            },
+            default: {
+                text: 'In progress',
+                colorClass: 'govuk-tag--blue',
+            },
+        };
+
+        const standardStatuses = {
+            Despatched: {
+                text: 'Despatched',
+                colorClass: '', // dark blue
+            },
+            default: {
+                text: casebookStatus,
+                colorClass: 'govuk-tag--blue',
+            },
+        };
+
+        if (!casebookStatus) {
+            return {
+                text: 'Not avialable',
```

Comment:
Available is spelt incorrectly

Reply:
Lol, can't believe no one else picked this up 🤦